frontend/src/components/camp/CampPrint.vue

```diff
@@ -29,18 +29,23 @@
         :title="result.title"
         class="mt-2" />
     </div>
+    <local-print-preview :config="config"
```

Comment:
Als hint für die, die das auch ausprobieren.
Das PDF Rendern wird automatisch gestartet, sobald man die Seite öffnet.
Im Firefox ist das Feedback für "PDF ist fertig gerendert" nicht so offensichtlich.

Im chrome sieht man dann das preview (aber nicht in den downloads), in firefox sieht man das pdf es nicht im preview, aber in den downloads